### PR TITLE
Mkdocs - commands and API

### PR DESCRIPTION
**Pull request by sheryjoe**

### CODE REVIEW

File: docs/workflow/groom.md [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
 
+Command Line Interface:  
 ``
 shapeworks readimage --name $1 antialias maxrmserror $2 iterations $3 layers $4 writeimage --name $5
 ``
 
+Python Interface:  
+```
+image = sw.Image(<input path>)
+image.antialis(iterations, maxRMSErr, layers).write(<output path>)
```
Comment (medakk) [inline docs/workflow/groom.md:18]: typo antialias

File: docs/dev/python-apis.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
 # How to Add Python APIs?
 
-![Under construction, coming soon!](../img/misc/under-construction.png)
\ No newline at end of file
+Python APIs in ShapeWorks are pybind wrappers of C++ functions.
```
Comment (iyerkrithika21) [inline docs/dev/python-apis.md:3]: Can you add details about the location of Python API files? Where can a new class/function be added?

Comment (jadie1) [reply docs/dev/python-apis.md:3]: I second this request, otherwise I think this PR looks good

File: docs/workflow/groom.md (hunk 1 above)
Comment (archanasri) [reply docs/workflow/groom.md]: Fixed

File: docs/dev/python-apis.md (hunk 2 above)
Comment (archanasri) [reply docs/dev/python-apis.md]: Updated it with filename.

**Review by jadie1 — APPROVED** [review]

**Review by medakk — APPROVED** [review]

**Review by iyerkrithika21 — APPROVED** [review]

**Comment by akenmorris** [comment]
This should probably be merged to release_6.0c or the changes in documentation will not be included in the release.

**Comment by archanasri** [comment]
@sheryjoe why are we not merging this to master?

**Comment by sheryjoe** [comment]
@archanasri I will submit another PR with the release notes and other documentation updates and target it to the release branch.

**Pull request closed without merging**